### DISCUSSION TITLE
is this repo still active?

### DISCUSSION THREAD

**Discussion by nikki-cat** [haltu/muuri#532] [General]
Just wondering. It looks nice, but I see that the activity if from last year :(

**Comment by niklasramo** [comment]
Yeah, Muuri is still maintained and actively developed. For the next version a lot of the work happens currently outside this repo though. The next version of Muuri will have the following dependencies which are (at the moment) mostly targeted for Muuri and also in active development:
* https://github.com/niklasramo/eventti
* https://github.com/niklasramo/tikki
* https://github.com/niklasramo/dragdoll
